src/download.jl

```diff
@@ -0,0 +1,49 @@
+"""
+    dataurl(dataset_code)
+
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""    
+
+function dataurl(dataset_provider, country_code)
+
+    # Only accepts GADM Dataset Provider
+    if dataset_provider != "GADM"
+        @error "Dataset Provider $dataset_provider not supported. Please try \"GADM/<Code>\"."
+    end
+    
+    @info "Fetching $country_code's data from GADM...\n"
+
+    "https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_$country_code_gpkg.zip"
```

Comment:
This is almost right. Check the general workflow in the previous comment. The dataurl function is general, and consumes any type of dataid, not only GADM. Inside this function you have a check to see if the dataid contains a GADM, and then you produce the URL.